nautobot/extras/tests/integration/test_configcontextschema.py

```diff
@@ -0,0 +1,69 @@
+from nautobot.utilities.testing.integration import SplinterTestCase
+
+
+class ConfigContextSchemaTestCase(SplinterTestCase):
+    """
+    Integration tests for the ConfigContextSchema model
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.user.is_superuser = True
+        self.user.save()
+        self.login(self.user.username, self.password)
+
+    def tearDown(self):
+        self.logout()
+        super().tearDown()
+
+    def test_create_valid_config_context_schema(self):
+        """
+        Given a clean slate, navigate to and fill out the form for a valid schema object
+        Assert the object is successfully created
+        And the user is redirected to the detail page for the new object
+        """
+        # Navigate to ConfigContextSchema list view
+        self.browser.visit(self.live_server_url)
+        self.browser.links.find_by_partial_text("Extensibility").click()
+        self.browser.links.find_by_text("Config Context Schemas").click()
+
+        # Click add add button
+        self.browser.find_by_xpath("/html/body/div/div[1]/a").click()
```

Comment:
It appears you copied the Xpath from developer tools, but this isn't quite readable. Should we find the button by text instead?

Reply:
As discussed, we will leave this as-is for now, and maybe we will learn of a better pattern with splinter in the future.